src/Simplify.elm

```diff
@@ -4368,16 +4368,22 @@ getReplaceAlwaysByItsResultFix : ModuleNameLookupTable -> Node Expression -> May
 getReplaceAlwaysByItsResultFix lookupTable expressionNode =
     case AstHelpers.removeParens expressionNode of
         Node _ (Expression.LambdaExpression lambda) ->
-            case List.map AstHelpers.removeParensFromPattern lambda.args of
-                (Node _ Pattern.AllPattern) :: [] ->
-                    Just
-                        (keepOnlyFix { parentRange = Node.range expressionNode, keep = Node.range lambda.expression })
+            case lambda.args of
+                firstArg :: argsAfterFirst ->
+                    case AstHelpers.removeParensFromPattern firstArg of
+                        Node allPatternRange Pattern.AllPattern ->
+                            case argsAfterFirst of
+                                [] ->
+                                    Just (keepOnlyFix { parentRange = Node.range expressionNode, keep = Node.range lambda.expression })
+
+                                (Node secondRange _) :: _ ->
+                                    Just
+                                        [ Fix.removeRange { start = allPatternRange.start, end = secondRange.start } ]
```

Comment:
Actually, the range we should be deleting from should be `Node.range firstArg`. If we have `\(_) b ->` then we are currently deleting `_) `.
Let's add a test with like `\(_) (b) ->` to not have this in the future.

Reply:
Very true. Fixed and added test.